ajax/libs/weather/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "weather",
+  "npmName": "weather.js",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "weather*"
+      ]
+    }
+  ],
+  "filename": "weather.min.js",
+  "version": "0.0.2",
+  "description": "Weather.js ==========",
```

Comment:
Please use something meaningful here, maybe the description on GitHub will be better (real weather for Javascript)`.